composer.json

```diff
@@ -12,6 +12,7 @@
         "drupal-scaffold": {
             "file-mapping": {
                 "[project-root]/wodby.yml": "assets/wodby.yml",
+                "[project-root]/README.md": "assets/README.md",
```

Comment:
That would overwrite the README every time drupal:scaffold runs. That needs to be prevented.